doc/ompl_constrained_planning/ompl_constrained_planning.rst

```diff
@@ -0,0 +1,85 @@
+OMPL Constrained Planning
+=========================
+.. image:: ompl_constrained_planning_header.png
+   :width: 600px
+
+This tutorial shows you how to use OMPL's `Constrained planning capabilities`_ from MoveIt. To illustrate the capabilities of this planner, three planning problems are solved with different types of path constraints. It will be easier to follow if you have read through the tutorial on the `move group Python interface`_.
+
+For a short walkthrough of running the tutorial, you can watch `this video`_.
+
+When can I use this planner?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The interface currently only supports position constraints on any link of the robot, where the constrained region is represented using a box_. The planning approach provides an alternative for the `enforce_joint_model_state_space`_ option. It is expected to be most valuable for constraint regions that have a small (or zero) volume in Cartesian space, where the rejection sampling does not always works. For example, keeping the end-effector on a plane or along a line.
```

Comment:
This is not true anymore as the tutorial contains an example with orientation constraints.

Reply:
should we split the tutorial in two PRs: 1 for position + 1 for orientation (+ eventually pose) ?
Or we keep one PR that will work partly after the moveit position PR is merged and entirely once the moveit orientation PR is merged?